src/main/java/chess/view/InputView.java

```diff
@@ -0,0 +1,11 @@
+package chess.view;
+
+import java.util.Scanner;
+
+public class InputView {
+    final static Scanner scanner = new Scanner(System.in);
```

Comment:
```suggestion
    private static final Scanner SCANNER = new Scanner(System.in);
```

Reply:
```
The Java Language Specification recommends listing modifiers in the following order:
1. Annotations
2. public
3. protected
4. private
5. abstract
6. static
7. final
8. transient
9. volatile
10. synchronized
11. native
12. strictfp
Not following this convention has no technical impact, but will reduce the code's readability because most developers are used to the standard order.
```